SetGame/SetGame/SetBoardView.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  SetBoardView.swift
+//  SetGame
+//
+//  Created by Nogah Melamed Cohen on 17/10/2019.
+//  Copyright © 2019 Nogah Melamed Cohen. All rights reserved.
+//
+
+import UIKit
+
+//board view with grid and SetCardViews as subviews
+class SetBoardView: UIView {
+    //grid for positioning cards on board
+    var boardGrid = Grid(layout: .dimensions(rowCount: 4, columnCount: 3)) { didSet { setNeedsDisplay(); setNeedsLayout() } }
+    
+    override func draw(_ rect: CGRect) {
+        boardGrid.frame = rect
+    }
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+    }
+
+}
+
+class SetCardView: UIView {
```

Comment:
I would prefer to have `SetCardView` in its own file `SetCardView.swift`. Usually the project code is easier to navigate this way. Only "private classes" should be in the same file as the public class.

On the positive side, excellent separation of code into methods in this class